### PR TITLE
docs: usage with immer

### PR DESCRIPTION
**Pull request by meluskyc**
Updating docs per #74 to warn when using auto freeze with immer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,15 @@ const Component = () => {
...
+## Usage with immer
+
+Disabling auto freeze is recommended. JavaScript does not support nested proxies of frozen objects.
+
+```js
+import { setAutoFreeze } from 'immer';
+setAutoFreeze(false);
+```
+
 *   [CodeSandbox](https://codesandbox.io/s/proxy-memoize-demo-yo00p)
```
Comment (dai-shi) [inline README.md:189]: I think we should move this line above the new section.

**Review by dai-shi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (meluskyc) [reply README.md:189]: OK, moved the new line up, if that's what you were looking for.

Comment (dai-shi) [reply README.md:188]: I mean this codesandbox link is for zustand usage.

Comment (meluskyc) [reply README.md]: Ah, thanks, sure fixed.

**Review by dai-shi — APPROVED** [review]
LGTM

**Pull request merged**